backend/api/src/test/kotlin/software/shonk/adapters/incoming/ShorkInterpreterControllerV1IT.kt

```diff
@@ -90,371 +89,443 @@ class ShorkInterpreterControllerV1IT : AbstractControllerTest() {
         return lobbiesArray.map { Json.decodeFromJsonElement<LobbyStatus>(it) }
     }
 
-    @Test
-    fun `test post and get player code`() = runTest {
-        client.post("/api/v1/lobby") {
-            contentType(ContentType.Application.Json)
-            setBody("{\"playerName\":\"playerA\"}")
-        }
-        client.post("/api/v1/lobby/0/code/playerA") {
-            contentType(ContentType.Application.Json)
-            setBody("{\"code\": \"someString\"}")
-        }
-
-        client.post("/api/v1/lobby/0/code/playerB") {
-            contentType(ContentType.Application.Json)
-            setBody("{\"code\": \"someOtherString\"}")
-        }
+    @Nested
+    inner class GetAndPostPlayerCode {
```

Comment:
I don't really like the naming of this class / scope.
Something like `Create and read back player code` or smth like that would be better imo